spec/models/average_ranking_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe Restaurant do
+  let(:restaurant) { Restaurant.create(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.